Application/EdFi.Ods.Api/Controllers/TokenController.cs

```diff
@@ -36,8 +38,60 @@ public TokenController(ITokenRequestProvider provider)
         [HttpPost]
         [AllowAnonymous]
         [Consumes("application/json")]
-        public async Task<IActionResult> Post([FromBody] TokenRequest tokenRequest)
+        public async Task<IActionResult> PostFromJsonAsync([FromBody] TokenRequest tokenRequest)
```

Comment:
Did you rename PostFromFrom => PostAsync?

Reply:
Fixed